app/forms/signin/passwordless_form.rb

```diff
@@ -0,0 +1,27 @@
+module Signin
+  class PasswordlessForm
+    include ActiveModel::Model
+
+    attr_accessor :user, :installation_id
+
+    validates :user, presence: true
+    validates :installation_id, presence: true
+
+    def submit
+      return if invalid?
+
+      assessment = IncogniaApi.instance.register_login(
+        account_id: user.account_id,
+        installation_id: installation_id,
+      )
+
+      return user if assessment.risk_assessment.to_sym == :low_risk
```

Comment:
It would be nice having a `low_risk?` method on the returned object. But it belongs to Incongia API, right?

Reply:
@julianalucena 
Agreed with the natural step you mentioned. 
I didn't suggest changing there because I thought it was too much for the current scope, but if we start using it more that would be a significant improvement.

Reply:
Good point!